Foundation/Data.swift

```diff
@@ -138,9 +138,9 @@ public final class _DataStorage {
     @discardableResult
     public func withUnsafeBytes<Result>(in range: Range<Int>, apply: (UnsafeRawBufferPointer) throws -> Result) rethrows -> Result {
         switch _backing {
-        case .swift: fallthrough
-        case .immutable: fallthrough
-        case .mutable:
+        case .swift,
```

Comment:
Let's skip the `Data` ones since we're in the process of merging the one from the overlay in here.

Reply:
Remove all changes to `Data.swift` file.